README.md

```diff
@@ -0,0 +1,17 @@
+# z2csl - Zotero to CSL extension and mappings
+
+This is a Zotero extension which creates up-to-date mappings from Zotero to CSL. Moreover, the current version of the mappings is hosted at  :point_right: **http://aurimasv.github.io/z2csl/typeMap.xml** :point_left: .
```

Comment:
Okay, I will change to `https` links, but I cannot change the link in the description:

![description](https://cloud.githubusercontent.com/assets/5199995/16360863/cbd5c0ac-3b78-11e6-9759-3ab288af4195.png)

Only you as the repo owner can change that link.